Droidcon-Boston/app/src/main/res/layout/nav_header.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="192dp"
+    android:theme="@style/ThemeOverlay.AppCompat.Dark">
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:scaleType="centerCrop"
+        android:src="@drawable/navigation_header_image" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom"
+        android:layout_marginBottom="18dp"
+        android:layout_marginEnd="18dp"
+        android:layout_marginStart="16dp"
+        android:text="@string/str_droidcon_boston_18"
+        android:textAppearance="@style/TextAppearance.AppCompat.Headline" />
```

Comment:
can we add the app logo or something to this nav header?

Reply:
Could you please clarify how you would like it to be.
Does this work?
![screen shot 2018-02-18 at 11 50 36 am](https://user-images.githubusercontent.com/2096087/36348977-10431d76-14a2-11e8-8258-2f59e6fd05e5.png)